modules/core/shared/src/main/scala/lucuma/core/math/Epoch.scala

```diff
@@ -53,60 +60,75 @@ final class Epoch private (val scheme: Epoch.Scheme, val toMilliyears: Int) {
     }
 
   override def hashCode: Int =
-    scheme.hashCode ^ toMilliyears
+    scheme.hashCode ^ toMilliyears.value
 
   override def toString: String =
-    Epoch.fromString.taggedToString("Epoch", this)
+    Epoch.fromString.asFormat.taggedToString("Epoch", this)
 
 }
 
 object Epoch extends EpochOptics {
+  type Year         = RefinedInterval.Closed[1900, 3000]
+  type MilliYear    = RefinedInterval.Closed[1900000, 3000999]
+  type IntYear      = Int Refined Year
```

Comment:
If the base unit is `IntMilliYear` would it make sens for this to be e.g. `RationalYear`?